packages/babel-parser/test/fixtures/core/uncategorised/108/output.json

```diff
@@ -0,0 +1,112 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 22,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 1,
+      "column": 22
+    }
+  },
+  "errors": [
+    "SyntaxError: Invalid regular expression flag (1:17)",
+    "SyntaxError: Invalid regular expression flag (1:19)",
+    "SyntaxError: Invalid regular expression flag (1:20)",
+    "SyntaxError: Invalid regular expression flag (1:21)",
+    "SyntaxError: Invalid regular expression flag (1:22)"
```

Comment:
The input file is `var x = /[P QR]/\u0067`. It throws one error for each character of `\u0067`, but I think that it is ok?